trie/trienode/node.go

```diff
@@ -39,7 +39,7 @@ func (n *Node) Size() int {
 
 // IsDeleted returns the indicator if the node is marked as deleted.
 func (n *Node) IsDeleted() bool {
-	return n.Hash == (common.Hash{})
+	return len(n.Blob) == 0
```

Comment:
L184-186 is different from OP

Reply:
The correct code is `len(n.Blob) == 0`.

Reply:
@jyc228 will mark L184-186 with Kroma diff sign.